lumapps/api/jwt.py

```diff
@@ -0,0 +1,80 @@
+import jwt
```

Comment:
Several things

- Jwt is not listed as a dependency in the pyproject.toml please add it via the command `poetry add jwt`
- Could you rename this file lumapps-jwt for clarirty as it concerns only lumapps jwt (not jwt in general)
- Please add one/several basic tests.
- Add a docstring describing the goal of the module